packages/devtools_shared/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ Copyright 2025 The Flutter Authors
 Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
 -->
+# 11.3.0
```

Comment:
@bkonyi does this need to be a major version bump in order to do a major version bump to the `dtd` dep?

Reply:
I don't think so. The breaking changes in the 3.0.0 changelog don't look like they'd break much code, if any. If you want to be extra safe, it wouldn't hurt.